Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -299,6 +299,18 @@
           "Name": "DoNotExposePlaintextSecrets/ConvertToSecureString",
           "ContentsRegex": "$SEC101/052.ConvertToSecureString",
           "MessageArguments": { "secretKind": "ConvertTo-SecureString powershell" }
+        },
+        {
+          "Id": "SEC101/053",
+          "Name": "DoNotExposePlaintextSecrets/GoogleServiceAccountKey",
+          "ContentsRegex": "$SEC101/053.GoogleServiceAccountKeyConsoleFormat",
+          "MessageArguments": { "secretKind": "Google Service Account Key" }
```

Comment:
>Google Service Account Key [](start = 47, length = 26)

please follow casing conventions for this code snippet. we generally uppercase brands only, everything else is as if rendered in a standard english sentence. and so i'd expect 'Google service account key' here.

'SqlCredential object initializer'

Reply:
Done

---
In reply to: [591971919](https://github.com/microsoft/sarif-pattern-matcher/pull/280#discussion_r591971919) [](ancestors = 591971919)